pos_partner_deselection/README.rst

```diff
@@ -2,11 +2,11 @@
    :target: https://www.gnu.org/licenses/lgpl
    :alt: License: LGPL-3
 
-==================
- Deselect Partner
-==================
+====================
+Partner  deselection
+====================
```

Comment:
https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#titles-and-sections

Reply:
+